src/lib/migrations/routines/index.ts

```diff
@@ -9,6 +10,7 @@ export interface Migration {
 	down?: (client: MongoClient) => Promise<boolean>;
 	runForFreshInstall?: "only" | "never"; // leave unspecified to run for both
 	runForHuggingChat?: "only" | "never"; // leave unspecified to run for both
+	runEveryTime: boolean;
```

Comment:
```suggestion
	runEveryTime?: boolean;
```

let's make it optional since it was not present initially